### PR TITLE
f-breadcrumbs@0.1.0 - generated component structure

### PR DESCRIPTION
**Pull request by matthewhardern**
New component to add bread crumbs, just the generated output from the generator. Modified sass options path until this is fixed in the generator.

### CODE REVIEW

**Comment by ashleynolan** [comment]
Super nit. I'd just call this package `f-breadcrumbs` (lose the extra dash) if that's easy enough to change

File: packages/components/molecules/f-breadcrumbs/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+
+v0.1.0
+------------------------------
+*January 20, 2021*
+
+### Added
+- Add your change under the following headings: `Added`/`Changed`/`Deprecated`/`Removed`/`Fixed`/`Security`
```
Comment (ashleynolan) [inline packages/components/molecules/f-breadcrumbs/CHANGELOG.md]: Should update this 😄

File: packages/components/molecules/f-breadcrumbs/README.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    ```bash
+    npm install @justeat/f-breadcrumbs
+    ```
+
+2.  Import the component
+
+    You can import it in your Vue SFC like this (please note that styles have to be imported separately):
+
+    ```
+    import BreadCrumbs from '@justeat/f-bread-crumbs';
```
Comment (ashleynolan) [inline packages/components/molecules/f-breadcrumbs/README.md]:
```suggestion
    import BreadCrumbs from '@justeat/f-breadcrumbs';
```

File: packages/components/molecules/f-breadcrumbs/README.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    import '@justeat/f-bread-crumbs/dist/f-bread-crumbs.css';
```
Comment (ashleynolan) [inline packages/components/molecules/f-breadcrumbs/README.md]:
```suggestion
    import '@justeat/f-breadcrumbs/dist/f-breadcrumbs.css';
```

File: packages/components/molecules/f-breadcrumbs/README.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+        components: {
+            BreadCrumbs
+        }
+    }
+    ```
+
+    If you are using Webpack, you can import the component dynamically to separate the `bread-crumbs` bundle from the main `bundle.client.js`:
+
+    ```
+    import '@justeat/f-bread-crumbs/dist/f-bread-crumbs.css';
```
Comment (ashleynolan) [inline packages/components/molecules/f-breadcrumbs/README.md]:
```suggestion
    import '@justeat/f-breadcrumbs/dist/f-breadcrumbs.css';
```

File: packages/components/molecules/f-breadcrumbs/README.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+
+    export default {
+        components: {
+            ...
+            BreadCrumbs: () => import(/* webpackChunkName: "bread-crumbs" */ '@justeat/f-bread-crumbs')
```
Comment (ashleynolan) [inline packages/components/molecules/f-breadcrumbs/README.md]:
```suggestion
            BreadCrumbs: () => import(/* webpackChunkName: "breadcrumbs" */ '@justeat/f-breadcrumbs')
```

File: packages/components/molecules/f-breadcrumbs/package.json [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=10.0.0"
+  },
+  "keywords": [
+    "fozzie"
+  ],
+  "scripts": {
+    "prepublishOnly": "yarn lint && yarn test && yarn build",
+    "build": "vue-cli-service build --target lib --name f-bread-crumbs src/index.js",
```
Comment (ashleynolan) [inline packages/components/molecules/f-breadcrumbs/package.json]:
```suggestion
    "build": "vue-cli-service build --target lib --name f-breadcrumbs src/index.js",
```

File: packages/components/molecules/f-breadcrumbs/src/assets/scss/common.scss [hunk 7]
```diff
@@ -0,0 +1,11 @@
+// Set $theme variable so that fozzie doesn't complain when vars are imported below
+// n.b. This variables isn't actually used (unless there are theme specific vars from fozzie)
+// Instead, the theme is handled via a data-attribute on the component (via the `@mixin theme()` below)
+$theme: 'jet' !default;
+
+@import '../../../../../../../node_modules/@justeat/fozzie/src/scss/fozzie';
```
Comment (ashleynolan) [inline packages/components/molecules/f-breadcrumbs/src/assets/scss/common.scss]:
```suggestion
@import  '~@justeat/fozzie/src/scss/fozzie';
```

File: packages/components/molecules/f-breadcrumbs/src/index.js [hunk 8]
```diff
@@ -0,0 +1,35 @@
+/**
+ * @overview Fozzie Bread Crumbs Component JS Wrapper
+ *
+ * @module f-bread-crumbs
```
Comment (ashleynolan) [inline packages/components/molecules/f-breadcrumbs/src/index.js]:
```suggestion
 * @module f-breadcrumbs
```

File: packages/components/molecules/f-breadcrumbs/src/components/BreadCrumbs.vue [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+    props: {
+        locale: {
+            type: String,
+            default: ''
+        }
+    },
+    data () {
+        const locale = globalisationServices.getLocale(tenantConfigs, this.locale, this.$i18n);
+        const localeConfig = tenantConfigs[locale];
+        const theme = globalisationServices.getTheme(locale);
```
Comment (ashleynolan) [inline packages/components/molecules/f-breadcrumbs/src/components/BreadCrumbs.vue]: Can delete this line – we no longer use this (removing it from generator now)

File: packages/components/molecules/f-breadcrumbs/src/components/BreadCrumbs.vue [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+
+        return {
+            copy: { ...localeConfig },
+            theme
```
Comment (ashleynolan) [inline packages/components/molecules/f-breadcrumbs/src/components/BreadCrumbs.vue]: Same here – can remove

File: packages/components/molecules/f-breadcrumbs/src/components/BreadCrumbs.vue [hunk 11]
```diff
@@ -0,0 +1,49 @@
+<template>
+    <div
+        :data-theme="theme"
```
Comment (ashleynolan) [inline packages/components/molecules/f-breadcrumbs/src/components/BreadCrumbs.vue]: And the same for this line

File: packages/components/molecules/f-breadcrumbs/README.md [hunk 12]
```diff
@@ -0,0 +1,102 @@
+
+<div align="center">
+  <h1>f-bread-crumbs</h1>
```
Comment (ashleynolan) [inline packages/components/molecules/f-breadcrumbs/README.md]:
```suggestion
  <h1>f-breadcrumbs</h1>
```

File: packages/components/molecules/f-breadcrumbs/README.md [hunk 13]
```diff
@@ -0,0 +1,102 @@
...
+
+  <p>Provides clickable paths back to previous pages</p>
+</div>
+
+---
+
+[![npm version](https://badge.fury.io/js/%40justeat%2Ff-bread-crumbs.svg)](https://badge.fury.io/js/%40justeat%2Ff-bread-crumbs)
+[![CircleCI](https://circleci.com/gh/justeat/fozzie-components.svg?style=svg)](https://circleci.com/gh/justeat/workflows/fozzie-components)
+[![Coverage Status](https://coveralls.io/repos/github/justeat/f-bread-crumbs/badge.svg)](https://coveralls.io/github/justeat/f-bread-crumbs)
+[![Known Vulnerabilities](https://snyk.io/test/github/justeat/f-bread-crumbs/badge.svg?targetFile=package.json)](https://snyk.io/test/github/justeat/f-bread-crumbs?targetFile=package.json)
```
Comment (ashleynolan) [inline packages/components/molecules/f-breadcrumbs/README.md]:
```suggestion
[![npm version](https://badge.fury.io/js/%40justeat%2Ff-breadcrumbs.svg)](https://badge.fury.io/js/%40justeat%2Ff-bread-crumbs)
[![CircleCI](https://circleci.com/gh/justeat/fozzie-components.svg?style=svg)](https://circleci.com/gh/justeat/workflows/fozzie-components)
[![Coverage Status](https://coveralls.io/repos/github/justeat/f-breadcrumbs/badge.svg)](https://coveralls.io/github/justeat/f-breadcrumbs)
[![Known Vulnerabilities](https://snyk.io/test/github/justeat/f-breadcrumbs/badge.svg?targetFile=package.json)](https://snyk.io/test/github/justeat/f-breadcrumbs?targetFile=package.json)
```

**Review by ashleynolan — CHANGES_REQUESTED** [review]

**Review by simonsandersje — DISMISSED** [review]

**Review by dandel10n — DISMISSED** [review]

File: packages/components/molecules/f-breadcrumbs/test/specs/accessibility/axe-accessibility.spec.js [hunk 14]
```diff
@@ -0,0 +1,8 @@
+import { getAccessibilityTestResults } from '../../../../../../../test/utils/axe-helper';
+
+describe('Accessibility tests', () => {
+    it('a11y - should test f-breadCrumbs component WCAG compliance', () => {
+        // Act
+        const axeResults = getAccessibilityTestResults('f-breadCrumbs');
```
Comment (ashleynolan) [inline packages/components/molecules/f-breadcrumbs/test/specs/accessibility/axe-accessibility.spec.js]: Just to check  – should this be `f-breadcrumbs` to match the `package.json` name, or is the camelCase ok?

Comment (matthewhardern) [reply packages/components/molecules/f-breadcrumbs/test/specs/accessibility/axe-accessibility.spec.js]: I am not sure tbh, @siggerzz is probably best to answer that question? if it needs changing can we also change in the generator.

Comment (siggerzz) [reply packages/components/molecules/f-breadcrumbs/test/specs/accessibility/axe-accessibility.spec.js]: Yeah, I'd rename this to f-breadcrumbs.

File: packages/components/molecules/f-breadcrumbs/src/components/BreadCrumbs.vue [hunk 15]
```diff
@@ -0,0 +1,46 @@
+<template>
+    <div
+        :class="$style['c-breadCrumbs']"
+        data-test-id="breadCrumbs-component">
```
Comment (siggerzz) [inline packages/components/molecules/f-breadcrumbs/src/components/BreadCrumbs.vue]: This should also be `breadcrumbs-component`

File: packages/components/molecules/f-breadcrumbs/test-utils/component-objects/f-breadCrumbs.component.js [hunk 16]
```diff
@@ -0,0 +1,5 @@
+const breadCrumbsComponent = () => $('[data-test-id="breadCrumbs-component"]');
```
Comment (siggerzz) [inline packages/components/molecules/f-breadcrumbs/test-utils/component-objects/f-breadCrumbs.component.js]: ditto

File: packages/components/molecules/f-breadcrumbs/test/specs/component/f-breadCrumbs.component.spec.js [hunk 17]
```diff
@@ -0,0 +1,13 @@
+import BreadCrumbsComponent from '../../../test-utils/component-objects/f-breadCrumbs.component';
+
+describe('f-breadCrumbs component tests', () => {
+    beforeEach(() => {
+        browser.url('/?path=/story/components--bread-crumbs-component');
```
Comment (siggerzz) [inline packages/components/molecules/f-breadcrumbs/test/specs/component/f-breadCrumbs.component.spec.js]: Change to `browser.url('/?path=/story/components--breadcrumbs-component');`

File: packages/components/molecules/f-breadcrumbs/test/specs/component/f-breadCrumbs.component.spec.js [hunk 18]
```diff
@@ -0,0 +1,13 @@
...
+        browser.switchToFrame(0);
+        BreadCrumbsComponent.waitForBreadCrumbsComponent();
```
Comment (siggerzz) [inline packages/components/molecules/f-breadcrumbs/test/specs/component/f-breadCrumbs.component.spec.js]: Change to `BreadcrumbsComponent.waitForBreadcrumbsComponent();`

File: packages/components/molecules/f-breadcrumbs/test/specs/component/f-breadCrumbs.component.spec.js [hunk 19]
```diff
@@ -0,0 +1,13 @@
...
+    });
+    it('should display the f-breadCrumbs component', () => {
```
Comment (siggerzz) [inline packages/components/molecules/f-breadcrumbs/test/specs/component/f-breadCrumbs.component.spec.js]: ditto

File: packages/components/molecules/f-breadcrumbs/test/specs/component/f-breadCrumbs.component.spec.js [hunk 20]
```diff
@@ -0,0 +1,13 @@
...
+        // Assert
+        expect(BreadCrumbsComponent.isBreadCrumbsComponentDisplayed()).toBe(true);
```
Comment (siggerzz) [inline packages/components/molecules/f-breadcrumbs/test/specs/component/f-breadCrumbs.component.spec.js]: ditto

File: packages/components/molecules/f-breadcrumbs/test/specs/component/f-breadCrumbs.component.spec.js [hunk 21]
```diff
@@ -0,0 +1,13 @@
+import BreadCrumbsComponent from '../../../test-utils/component-objects/f-breadCrumbs.component';
```
Comment (siggerzz) [inline packages/components/molecules/f-breadcrumbs/test/specs/component/f-breadCrumbs.component.spec.js]: Change filename to f-breadcrumbs.component

File: packages/components/molecules/f-breadcrumbs/stories/BreadCrumbs.stories.js [hunk 22]
```diff
@@ -0,0 +1,26 @@
+// Uncomment the import below to add prop controls to your Story (and add `withKnobs` to the decorators array)
+// import {
+//     withKnobs, select, boolean
+// } from '@storybook/addon-knobs';
+import { withA11y } from '@storybook/addon-a11y';
+import BreadCrumbs from '../src/components/BreadCrumbs.vue';
```
Comment (siggerzz) [inline packages/components/molecules/f-breadcrumbs/stories/BreadCrumbs.stories.js]: Change var name + Vue component to `Breadcrumbs`

File: packages/components/molecules/f-breadcrumbs/test/specs/accessibility/axe-accessibility.spec.js (hunk 14 above)
Comment (matthewhardern) [reply packages/components/molecules/f-breadcrumbs/test/specs/accessibility/axe-accessibility.spec.js]: done ditto below

File: packages/components/molecules/f-breadcrumbs/src/components/Breadcrumbs.vue [hunk 23]
```diff
@@ -0,0 +1,46 @@
+<template>
+    <div
+        :class="$style['c-breadCrumbs']"
```
Comment (ashleynolan) [inline packages/components/molecules/f-breadcrumbs/src/components/Breadcrumbs.vue]: Sorry to be a pain – is it worth getting rid of the camel casing everywhere to keep it consistent with the changes made for the tests? So just updating to `breadcrumbs` or `Breadcrumbs` everywhere then.

Comment (matthewhardern) [reply packages/components/molecules/f-breadcrumbs/src/components/Breadcrumbs.vue]: agreed pushed a change

File: packages/components/molecules/f-breadcrumbs/test/specs/accessibility/axe-accessibility.spec.js (hunk 14 above)
Comment (ashleynolan) [reply packages/components/molecules/f-breadcrumbs/test/specs/accessibility/axe-accessibility.spec.js]: Should this be `f-breadcrumbs` now?

**Review by ashleynolan — APPROVED** [review]

**Review by kevinrodrigues — APPROVED** [review]

**Pull request merged**